src/js/Editor.js

```diff
@@ -3,6 +3,9 @@
  */
 class Editor {
     constructor(id, language) {
+        if (language === 'java') {
```

Comment:
I wish there was a way to verify the language to outside Editor.js.

Reply:
modified